src/components/TodoModal/TodoModal.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Loader } from '../Loader';
+import { Todo } from '../../type/Todo';
+import { actions as currentTodoActions } from '../../store/currentTodo';
+import { useAppSelector } from '../../store';
+import { User } from '../../type/User';
+import { getUser } from '../../api';
+
+type Props = {
+};
+
+export const TodoModal: React.FC<Props> = () => {
+  const [user, setUser] = useState<User | null>(null);
+  const currentTodo: Todo | null = useAppSelector(state => state.currentTodo);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (currentTodo?.userId) {
+      getUser(+currentTodo.userId).then(setUser);
+    }
+  }, []);
+
+  return (
+
+    <div className="modal is-active" data-cy="modal">
+      <div className="modal-background" />
+
+      {!user ? (
+        <Loader />
+      ) : (
```

Comment:
```suggestion
      {!user 
        ? <Loader />
        : (
```

Same problem as for me. It's hard to read.